### ISSUE TITLE
pre loss is

### ISSUE THREAD

**Issue by zrx0311** [yijingru/BBAVectors-Oriented-Object-Detection#83]

**Comment by yijingru** [comment]
Maybe the batch size is too small.

**Comment by zrx0311** [comment]
thanks！And if the BatchSize is set to 10, what do you think is appropriate for the learning rate

**Comment by zrx0311** [comment]
thank！Now i have 4 Tesla.Would you get a better result if you didn't use BN?

**Comment by zrx0311** [comment]
thank！Now i have 4 Tesla.And would you get a better result if you didn't use BN?

**Comment by yijingru** [comment]
I haven't tried. But BN may enable more stable training or improve the generalization ability.
